Testscripts/Linux/SetupHbKernel.sh

```diff
@@ -96,16 +95,28 @@ function Main() {
 		LogMsg "$?: Installed required packages, $req_pkg"
 
 		# Start kernel compilation
-		LogMsg "Clone and compile new kernel from $hb_url to /usr/src/linux"
-		git clone $hb_url /usr/src/linux
-		LogMsg "$?: Cloned the kernel source repo in /usr/src/linux"
+		LogMsg "Cloning a new kernel from $hb_url to $linux_path"
+		git clone $hb_url $linux_path
+		LogMsg "$?: Cloned the kernel source repo in $linux_path"
 
-		cd /usr/src/linux/
+		cd $linux_path
 
 		git checkout $hb_branch
 		LogMsg "$?: Changed to $hb_branch"
 
-		cp /boot/config*-azure /usr/src/linux/.config
+		if [[ "$DISTRO" =~ "redhat" || "$DISTRO" =~ "centos" ]];then
+			cp /boot/config* $linux_path/.config
+			# Commented out CONFIG_SYSTEM_TRUSTED_KEY parameter for redhat kernel compilation
+			sed -i -e "s/CONFIG_MODULE_SIG_KEY=/#CONFIG_MODULE_SIG_KEY=/g" $linux_path/.config
```

Comment:
Will modifying the kernel .config be a problem for other tests that are executed after this one?

Reply:
You meant "UseExistingRG' option? It will provision new VM inside if I recall correctly.